src/status_im2/contexts/syncing/standard_authentication/enter_password.cljs

```diff
@@ -0,0 +1,97 @@
+(ns status-im2.contexts.syncing.standard-authentication.enter-password
+  (:require [utils.i18n :as i18n]
+            [quo2.core :as quo]
+            [quo2.foundations.colors :as colors]
+            [react-native.core :as rn]
+            [status-im2.contexts.onboarding.common.background.view :as background]
+            [utils.re-frame :as rf]
+            [clojure.string :as string]
+            [utils.security.core :as security]
+            [status-im2.contexts.syncing.standard-authentication.style :as style]
+            [status-im2.contexts.syncing.standard-authentication.enter-password :as enter-password]))
+
+(defn- get-error-message
+  [error]
+  (if (and (some? error)
+           (or (= error "file is not a database")
+               (string/starts-with? error "failed to set ")
+               (string/starts-with? error "Failed")))
+    (i18n/label :t/oops-wrong-password)
+    error))
+
+(defn view
+  [set-code button-label]
+  (let [entered-password                    (atom "")
+        {:keys [key-uid display-name
+                customization-color]}       (rf/sub [:profile/multiaccount])
+        {:keys [error processing password]} (rf/sub [:profile/login])
+        sign-in-enabled?                    (rf/sub [:sign-in-enabled?])
+        profile-picture                     (rf/sub [:profile/login-profiles-picture key-uid])
+        error                               (get-error-message error)]
+    [background/view true]
+    [:<>
+     [rn/view {:style style/enter-password-container}
+      [rn/view
+       [quo/text
+        {:accessibility-label :sync-code-generated
+         :weight              :bold
+         :size                :heading-1
+         :style               {:color         (colors/theme-colors colors/neutral-100 colors/white)
+                               :margin-bottom 4}}
+        (i18n/label :t/enter-password)]
+
+       [rn/view
+        {:flex-direction :row :margin-bottom 20 :margin-top 8}
```

Comment:
move these to dedicated style file please.
ref : https://github.com/status-im/status-mobile/blob/develop/doc/new-guidelines.md#component-styles